src/components/auth/signin/SignInContainer.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from 'react'
+import { useAuthStore } from '../store/AuthStore'
+import Modal from '~/components/modal/Modal.jsx'
+import ConfirmModal from '../confirmModal/ConfirmModal'
+import styles from './SignInContainer.module.scss'
+
+export default function SignInContainer() {
+  const { email, setEmail } = useAuthStore()
+  const { password, setPassword } = useAuthStore()
+  const { switchToRegister } = useAuthStore()
+  const { login } = useAuthStore()
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false)
+  const [showConfirm, setShowConfirm] = useState(false)
```

Comment:
rename `showConfirm` to `isDirty` - this is common Front-End pattern for use cases like this.